README.markdown

```diff
@@ -115,6 +131,30 @@ One can apply a "sort" constraint, which analyzes documents in the specified ord
 
     $ mongo test --eval "var collection = 'users', sort = { updated_at : -1 }" variety.js
 
+### Output formats ###
+
+Variety supports two different output formats:
+
+- ASCII: nice formatted tables (as in this readme)
+- JSON: valid JSON results for subsequent processing in other tools (see also [quiet option](https://github.com/variety/variety#quiet-option))
+
+Default format is ```ascii```. You can select the type of format with property ```outputFormat``` provided to variety. Valid values are ```ascii``` and ```json```.
+
+    $ mongo test --quiet --eval "var collection = 'users', outputFormat=json" variety.js
+
+### Quiet option ###
+Both MongoDB and Variety outputs some additional information to standard output. If you want to remove this info, you can use ```--quiet``` option provided to ```mongo``` executable.
+Variety can also read that option and mute unnecessary output. This is useful in connection with ```outputFormat=json```. You receive then only JSON without any other characters around it.
```

Comment:
Correct English would be "You would then receive only JSON, without any other characters around it."